src/style/content/preferences.less

```diff
@@ -212,7 +194,7 @@ body.theme-dark {
   height: 1px;
   border: 0;
   margin: 22px 0 22px;
-  background: @gray-50;
+  background: var(--gray-50);
```

Comment:
this one can stay, I'll assume we want it right in the middle for both mode :sweat_smile:

Reply:
of for sure, I'm just saying I'm not going to protest against `var(--gray-50)` instead of `var(--new-css-variable-we'll-use-only-once)